task/s2i-angular-nginx/0.1/s2i-angular-nginx.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/displayName: s2i angular nginx
+    tekton.dev/categories: Image Build
+    tekton.dev/tags: image, build, angular, nginx
+    tekton.dev/pipelines.minVersion: '0.19'
```

Comment:
any reason for keeping pipelines.minVersion as 0.19 instead of 0.12? (folks won't be able to install this Task via tkn-hub CLI if they are using Tekton Pipelines older that 0.19)